pkg/mongoproxy/plugins/schema/schema.go

```diff
@@ -122,30 +125,15 @@ func (p *SchemaPlugin) Configure(d bson.D) error {
 
 	go func() {
 		for {
-			select {
-			case event, ok := <-watcher.Events:
-				if !ok {
-					return
-				}
-				if event.Name != p.conf.SchemaPath {
-					continue
-				}
-				logrus.Debugf("Schema watcher event: %v", event)
-				p.LoadSchema()
-
-			case err, ok := <-watcher.Errors:
-				if !ok {
-					return
-				}
-				logrus.Errorf("Schema watcher: %v", err)
-			}
+			p.LoadSchema()
+			time.Sleep(5 * time.Minute)
 		}
 	}()
 
 	if err := watcher.Add(path.Dir(p.conf.SchemaPath)); err != nil {
```

Comment:
I guess we dont need watcher, can we clean it up from codes?

Reply:
fixed